weighted_exclusivity_test/saddlepoint.py

```diff
@@ -46,7 +45,7 @@ def enumeration(k):
 
     return states, indices, gradient_indices, hessian_indices
 
-def saddlepoint(observed_t, observed_y, probabilities, enumeration_k=None):
+def saddlepoint(observed_t, observed_y, probabilities, verbose=False, enumeration_k=None):
```

Comment:
We do not pass `verbose` as an argument to `saddlepoint` in `exclusivity_tests.py`.  Also, let's remove `enumeration_k` as an argument because we do not use it.